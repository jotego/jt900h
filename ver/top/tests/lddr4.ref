00000011 - 00000000 - 00000000 - 00000000
00000000 - 00000000 - 00000000 - 00000000
00000076 - 00000000 - 00000000 - 00000000
0000babe - 00000000 - 00000000 - 00000000
-- index registers --
00000076 - 00000056 - 00000000 - 00000000
SR = f842
