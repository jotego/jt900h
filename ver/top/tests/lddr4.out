00000011 - 00000000 - 00000000 - 00000000
00000000 - 00000000 - 00000000 - 00000000
0000006c - 00000000 - 00000000 - 00000000
0000babe - 00000000 - 00000000 - 00000000
-- index registers --
0000006c - 0000004c - 00000000 - 00000000
SR = 42f8
